frontend/App.js

```diff
@@ -1,21 +1,53 @@
-import { StatusBar } from 'expo-status-bar';
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import * as React from 'react';
+import { NavigationContainer } from '@react-navigation/native';
+import { createStackNavigator } from '@react-navigation/stack';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import { MaterialCommunityIcons, Ionicons, FontAwesome5 } from '@expo/vector-icons';
+import { DiagnosticsTab, DataTab, HomeTab, AmbianceTab, Start } from './homeTabs';
+import SettingsTab from './settingsTab';
 
-export default function App() {
+function HomeTabs() {
   return (
-    <View style={styles.container}>
-      <Text>Sleep thingy holder text</Text>
-      <StatusBar style="auto" />
-    </View>
+    <Tab.Navigator initialRouteName="Home" screenOptions={({ route }) => ({
+      tabBarIcon: ({ focused, color, size }) => {
+        switch (route.name) {
+          case "Diagnostics":
+            return <MaterialCommunityIcons name="hospital" size={size} color={color} />;
+          case "Data":
+            return <MaterialCommunityIcons name="waveform" size={size} color={color} />;
+          case "Home":
+            return <Ionicons name="ios-globe-outline" size={size} color={color} />;
+          case "Ambiance":
+            return <FontAwesome5 name="lightbulb" size={size} color={color} />;
+          case "Settings":
+            return <Ionicons name="settings-outline" size={size} color={color} />;
+          default:
+            return;
+        }
+      }
+    })}>
+      <Tab.Screen name="Diagnostics" component={DiagnosticsTab} />
+      <Tab.Screen name="Data" component={DataTab} />
+      <Tab.Screen name="Home" component={HomeTab} />
+      <Tab.Screen name="Ambiance" component={AmbianceTab} />
+      <Tab.Screen name="Settings" component={SettingsTab} />
+    </Tab.Navigator>
   );
 }
 
-const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    backgroundColor: '#fff',
-    alignItems: 'center',
-    justifyContent: 'center',
-  },
-});
+const Stack = createStackNavigator();
+
+const Tab = createBottomTabNavigator();
+
+function App() {
+  return (
+    <NavigationContainer>
+      <Stack.Navigator initialRouteName="Home">
+        <Stack.Screen name="Start" component={Start} />
+        <Stack.Screen name="Home" component={HomeTabs} options={{ headerShown: false }} />
+      </Stack.Navigator>
+    </NavigationContainer>
```

Comment:
Nested the Tab Navigator inside the Stack Navigator as per [React Navigation - hiding tab bar in specific screens](https://reactnavigation.org/docs/hiding-tabbar-in-screens)

This way, the Start screen does not display the bottom tab bar.